hera_mc/cm_transfer.py

```diff
@@ -247,14 +249,12 @@ def db_validation(maindb_pw, session):
     if not is_maindb:
         return True
 
-    allowed = True
     if maindb_pw is False:
-        print('Error:  attempting access to main db without a password')
-        allowed = False
-    elif maindb_pw != 'pw4maindb':
-        print('Error:  incorrect password for main db')
-        allowed = False
-    return allowed
+        raise ValueError('Error:  attempting access to main db without a password')
+    if maindb_pw != 'pw4maindb':
```

Comment:
Is this actually a password in plain text? Do we want that here?

Reply:
We did talk about this long ago.  It is really to (a) prevent a whoopsie on (mostly) my part (b) stop people who don't know the system to get in.  It is a security thing, just a speed bump.

Reply:
ok. A speed bump is sensible.